weather_lookup/styles.css

```diff
@@ -35,14 +74,42 @@
   font-size: small;
 }
 
-#header > h2   { grid-area: 1 / 1 / 2 / 4; }
-#city-selector { grid-area: 2 / 2 / 3 / 3; }
-#prev_city_btn { grid-area: 2 / 1 / 3 / 2; }
-#rnd_city_btn  { grid-area: 2 / 3 / 3 / 4; }
 
 button { width: 200px; }
 button > div {
   text-overflow: ellipsis;
   overflow: hidden;
   white-space: nowrap;
+}
+
+/* Places the source link in upper right of page */
+#source_link {
+  position: fixed;
+  top: 5px;
+  right: 5px;
+  font-size: 0.9rem;
+  font-weight: lighter;
+  padding: 0;
+}
+
+#control_thresholds {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 500px));
+  grid-gap: 2rem;
+  justify-content: center;
+  justify-items: center;
+  align-items: center;
+}
+
+#control_thresholds > div {
+  font-size: 0.85rem;
+  color: #515151;
+}
+
+span.cold {
+  color: blue;
+}
+
+span.hot {
+  color: blue;
 }
```

Comment:
It might be worth converting this `.css` to `.scss` and using `theme %>% bs_add_rules(sass_file())`  to include it. Then hard coded colors like this could key off of the relevant Sass variables.

(Also, btw, you might want to consider doing this sort of thing via `.bg-*` utility classes? You can create you own classes if you need to https://rstudio.github.io/bslib/articles/theming.html#create-new-utility-classes)